package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fp-ts",
-  "version": "2.4.1",
+  "version": "2.5.0",
```

Comment:
Why do you bump version in this PR?

Reply:
mhh.. why not? new features -> bump version

Reply:
I'm just unused to such approach, ok :)